cmd/influx/cli/cli.go

```diff
@@ -198,13 +198,20 @@ func (c *CommandLine) Run() error {
 	c.Version()
 
 	// Only load/write history if HOME environment variable is set.
+	var historyDir string
 	if homeDir := os.Getenv("HOME"); homeDir != "" {
-		// Attempt to load the history file.
-		c.historyFilePath = filepath.Join(homeDir, ".influx_history")
-		if historyFile, err := os.Open(c.historyFilePath); err == nil {
-			c.Line.ReadHistory(historyFile)
-			historyFile.Close()
-		}
+		historyDir = homeDir
+	} else if userDir := os.Getenv("USERPROFILE"); userDir != "" {
+		historyDir = userDir
+	} else {
+		historyDir = os.TempDir()
```

Comment:
I think we should avoid this as it becomes a potential security hole. If a password gets put into a command such as from create user, I don't know if that's redacted in the history file and it probably isn't. The file in the temp folder would make it open to everyone on the system.

I would rather we just don't open the history file at all in that case and don't save history. Maybe warn the user that's what we're doing.